### PR TITLE
Grammar fixes

### PR DESCRIPTION
**Pull request by emmaodia**

### CODE REVIEW

File: docs/types/value-types.rst [hunk 1]
```diff
@@ -221,7 +221,7 @@ Operators:
 
 .. warning::
     If you convert a type that uses a larger byte size to an ``address``, for example ``bytes32``, then the ``address`` is truncated.
-    To reduce conversion ambiguity version 0.4.24 and higher of the compiler force you make the truncation explicit in the conversion.
+    To reduce conversion ambiguity version 0.4.24 and higher versions of the solidity compiler force you to make the truncation explicit in the conversion.
```
Comment (cameel) [inline docs/types/value-types.rst:224]:
```suggestion
    To reduce conversion ambiguity version 0.4.24 and higher versions of the Solidity compiler force you to make the truncation explicit in the conversion.
```

**Review by cameel — COMMENTED** [review]
Thanks for the PR!

Just one tiny correction. Also, please squash your commits and rebase on `develop` (instead of merging it into your PR) so that we can merge it.

**Comment by emmaodia** [comment]
Thank you, @cameel, for the feedback. I have duly noted it.
I will close out the PR and send in a new one.

**Comment by cameel** [comment]
But the PR was fine. It would have been enough to simply rebase it...

**Comment by emmaodia** [comment]
My mistake then. I have duly noted the corrections. Thanks again.

**Pull request closed without merging**